mk/meson/ppc64.cfg

```diff
@@ -0,0 +1,11 @@
+[built-in]
+c_args = ['']
+c_link_args = ['']
+cpp_args = ['']
+cpp_link_args = ['']
+
+[host_machine]
+system = 'linux'
+cpu_family = 'ppc64'
+cpu = 'ppc64'
+endian = 'little'
```

Comment:
the active ppc archs (ppc853x and qoriq) are both 32 bit, so I expect ppc32 here.
and the endian is 'big' not 'little'.

Reply:
now changed.